addons/sale/migrations/13.0.1.1/post-migration.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Payam Yasaie <https://www.tashilgostar.com>
+# Copyright 2020 ForgeFlow <http://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def update_account_move_team_partner_ids(cr):
+    openupgrade.logged_query(
+        cr, """
+            UPDATE account_move ammi
```

Comment:
```suggestion
            UPDATE account_move am
```
To avoid
```
psycopg2.ProgrammingError: missing FROM-clause entry for table "am"
LINE 5:             WHERE am.old_invoice_id = ai.id
```